pkg/pgmodel/metrics/database/metrics.go

```diff
@@ -0,0 +1,187 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/timescale/promscale/pkg/util"
+)
+
+var (
+	dbHealthErrors = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: util.PromNamespace,
+			Subsystem: "sql_database",
+			Name:      "health_check_errors_total",
+			Help:      "Total number of database health check errors.",
+		},
+	)
+	up = prometheus.NewGauge(
```

Comment:
Just 2 cents. Metric name should stay as is since this is aligned with the convention. I agree with Niksa that variable name could be better to allow easier understanding of the code base.

Maybe:
```suggestion
	upMetric = prometheus.NewGauge(
```
?